django/db/models/sql/query.py

```diff
@@ -1081,9 +1081,13 @@ def build_lookup(self, lookups, lhs, rhs):
             return
 
         lookup = lookup_class(lhs, rhs)
-        # Interpret '__exact=None' as the sql 'is NULL'; otherwise, reject all
-        # uses of None as a query value.
         if lookup.rhs is None:
+            # It should be possible to use 'None` for transforms
+            # For example it makes possible to query 'null' in JSON fields
+            if lookup.__class__.__name__ == 'JSONExact':
```

Comment:
Hello @timgraham !
Could you, please advice here? Would it be better to add some special attribute to lookup class and check it here instead of class name? 
Thanks